src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java

```diff
@@ -149,4 +145,35 @@ public void findAllTest() {
 
         assertEquals(genericEntities, foundEntities);
     }
+
+    @Test
+    public void getOpenHoursByPlaceTest() {
+        Place place = Place.builder().build();
+
+        List<OpeningHours> genericOpeningHours =
+            new ArrayList<>(Arrays.asList(new OpeningHours(), new OpeningHours()));
+
+        when(openHoursRepo.findAllByPlace(any())).thenReturn(genericOpeningHours);
+
+        List<OpeningHours> foundOpeningHours = openHoursService.getOpenHoursByPlace(place);
+
+        assertEquals(genericOpeningHours, foundOpeningHours);
+        Mockito.verify(openHoursRepo, times(1)).findAllByPlace(any());
+    }
```

Comment:
Please consider about removing some blank lines here and in **findAllByPlaceIdTest**(), now it's to much and it`s not good for big files where you will need to scroll all of this lines.
Consider about better grouping lines, for e.x:

Place place = Place.builder().build();
List<OpeningHours> genericOpeningHours = new ArrayList<>(Arrays.asList(new OpeningHours(), new OpeningHours()));
List<OpeningHours> foundOpeningHours = openHoursService.getOpenHoursByPlace(place);

 when(openHoursRepo.findAllByPlace(any())).thenReturn(genericOpeningHours);

 assertEquals(genericOpeningHours, foundOpeningHours);
 Mockito.verify(openHoursRepo, times(1)).findAllByPlace(any());

And at line **154** better to use just Arrays.asList(new OpeningHours(), new OpeningHours()), without new ArrayList()

Reply:
Actually I've done this in such way for more readability but I got you. And also when I put foundOpeningHours under genericOpeningHours test fails and the same case with set in method findAllByPlaceId.
Btw, thanks for useful hints.